docs/graphman.md

```diff
@@ -362,3 +363,56 @@ Inspect a block range, deleting any duplicated blocks:
 Inspect all blocks after block `13000000`:
 
     graphman --config config.toml chain check-blocks mainnet by-range --from 13000000
+
+<a id="chain-call-cache-remove"></a>
+# ⌘ Call Cache
+
+### SYNOPSIS
+
+Remove the call cache of the specified chain.
+
+If block numbers are not mentioned in `--from` and `--to`, then all the call cache will be removed.
+
+USAGE:
+    graphman chain call-cache <CHAIN_NAME> remove [OPTIONS]
+
+OPTIONS:
+    -f, --from <FROM>
+            Starting block number
+
+    -h, --help
+            Print help information
+
+    -t, --to <TO>
+            Ending block number
+
+### DESCRIPTION
+
+Remove the call cache of a specified chain.
+
+### OPTIONS
+
+The `from` and `to` options can be used to decide the block range of the call cache that needs to be removed.
+
+#### `from`
+   
+    graphman --config config.toml chain call-cache ethereum remove --from 10
+
+#### `to`
+
+    graphman --config config.toml chain call-cache ethereum remove --to 10
+
+#### `from` and `to`
+
+    graphman --config config.toml chain call-cache ethereum remove --from 10 --to 50
+
```

Comment:
The `OPTIONS` section should be dedicated only to explaining what each option does and how it affects the program execution. 

Instead of the examples above, we should describe how `--to` and `--from` affect block range selection. 
For instance, a reader might be interested in whether the `--from` block is included in the range or not (_"Are the range bounds inclusive?"_) and so on.

I believe those literal examples would best fit the `EXAMPLES` section, each preceded by a brief paragraph describing the expected outcome of that invocation.